app/models/question.rb

```diff
@@ -7,4 +7,6 @@ class Question < ApplicationRecord
 
   validates :name, presence: true, uniqueness: true
 
+  scope :active, -> { where(active: true).order('RANDOM()') }
```

Comment:
I do not think we need to randomise here

```suggestion
  scope :active, -> { where(active: true) }
```

Reply:
please suggest then how are we going to keep track which question to send in response. We have send only one question.

Reply:
We just need to return a question that is active and the user has not seen. So if our sample is `Question.where(active: true).not_seen.first` that will essentially be. Also `BY RANDOM()` in sql databases is less performant since it fetches all the rows and picks on at random.

Reply:
Yes, I am working on the logic that will get the question from the DB which has not been answered the user.